suite-native/module-device-onboarding/src/screens/ThpConfirmationScreen.tsx

```diff
@@ -27,12 +28,14 @@ export const ThpConfirmationScreen = ({
     }, [thpStep, navigation]);
 
     return (
-        <DeviceOnboardingScreenWithExitButton
+        <NonClosableDeviceOnboardingScreen
             isScrollable={false}
             noBottomPadding={true}
             hasBottomInset={false}
         >
-            <ContinueOnTrezorScreenContent />
-        </DeviceOnboardingScreenWithExitButton>
+            <Box flex={1}>
```

Comment:
This wrapping has no effect, may be reverted.

Reply:
Done